hackathon_site/dashboard/frontend/src/slices/event/teamAdminSlice.test.ts

```diff
@@ -0,0 +1,66 @@
+import store, { makeStore, RootState } from "../store";
+import {
+    teamAdminReducerName,
+    initialState,
+    teamAdminSliceSelector,
+    isLoadingSelector,
+    getAllTeams,
+    teamAdminSelectors,
+} from "./teamAdminSlice";
+import { makeMockApiListResponse, waitFor } from "../../testing/utils";
+import { mockTeams } from "../../testing/mockData";
+import { get } from "../../api/api";
+
+jest.mock("api/api", () => ({
+    ...jest.requireActual("api/api"),
+    get: jest.fn(),
+}));
+const mockedGet = get as jest.MockedFunction<typeof get>;
+
+const mockState: RootState = {
+    ...store.getState(),
+    [teamAdminReducerName]: initialState,
+};
+
+describe("Selectors", () => {
+    test("teamAdminSliceSelector returns the isLoading state", () => {
```

Comment:
change test name to "teamAdminSliceSelector returns teamAdminSlice"

Reply:
alright